extensions/blocks/podcast-player/icons/episode-playing-icon.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * External dependencies
+ */
+import { Path, SVG } from '@wordpress/components';
+
+export default (
+	<SVG height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Add `role="img"` and an `aria-labelledby="jetpack-podcast-player__playing-icon"` to the `<SVG` line, and directly beneath, add `<title id="jetpack-podcast-player__playingr-icon">{ __( 'Playing' ) }</title>` to have "Playing" announced for this icon with screen readers.

Reply:
I noticed that there's an `aria-hidden="true"` added by default to the `SVG` component: 

![image](https://user-images.githubusercontent.com/1451471/77531108-b2119a00-6e92-11ea-9a66-4e8f150a4ea1.png)

Does it conflict with the `role="img" aria-labelledby="..."` attributes we've just added?

Reply:
Hmm... Yeah, because `aria-hidden` indicates to the accessibility API that this should be ignored. Does setting `aria-hidden={ undefined }` on the `<SVG` work to remove it?

Reply:
Unfortunately, that attribute is hard-coded https://github.com/WordPress/gutenberg/blob/master/packages/primitives/src/svg/index.js#L29 :/

Reply:
Hrmmm.... I wonder what the intention is there. I guess the alternative is to just use `<VisuallyHidden>` text.

Reply:
Updated in https://github.com/Automattic/jetpack/pull/15101/commits/bd45c78466addc8bd5b59474201b6b7cbe1dfbfb.

Opened a PR in Gutenberg for custom SVG `aria-hidden` support: https://github.com/WordPress/gutenberg/pull/21173

As agreed in p1585170298230600-slack-ajax, we will not make any further changes in this matter here and will follow up once the Gutenberg PR gets reviewed & merged/closed.